Common/MVKOSExtensions.mm

```diff
@@ -97,16 +97,20 @@ bool mvkGetEnvVarBool(std::string varName, bool* pWasFound) {
 #pragma mark System memory
 
 uint64_t mvkGetSystemMemorySize() {
+#if MVK_MACOS || MVK_IOS
 	mach_msg_type_number_t host_size = HOST_BASIC_INFO_COUNT;
 	host_basic_info_data_t info;
 	if (host_info(mach_host_self(), HOST_BASIC_INFO, (host_info_t)&info, &host_size) == KERN_SUCCESS) {
 		return info.max_mem;
 	}
+#endif
```

Comment:
This triggers a `Control may reach end of non-void function` error during compilation.

Needs a second copy of `return 0;` before leaving the `#if MVK_MACOS || MVK_IOS` block.

As we've discussed, leave the `#if MVK_TVOS` in place. Don't remove it to provide a single `return 0;` fallback for all platforms. 😉

Reply:
Oops, nice catch!